packages/react-strict-dom/babel/index.js

```diff
@@ -204,4 +212,44 @@ module.exports = function ({ types: t }, options = {}) {
       }
     }
   };
+}
+
+const defaultOptions = {
+  dev: true,
+  debug: true,
+  rootDir: process.cwd()
 };
+
+function reactStrictPreset(_, options = {}) {
+  const opts = { ...defaultOptions, ...options };
+
+  return {
+    plugins: [
+      [
+        reactStrictPlugin,
+        {
+          debug: opts.debug
+        }
+      ],
+      [
+        styleXPlugin,
+        {
+          dev: opts.dev,
+          importSources: [{ from: 'react-strict-dom', as: 'css' }],
+          runtimeInjection: opts.dev, // temporary until Expo/Metro can support built-time
+          styleResolution: 'property-specificity',
+          unstable_moduleResolution: {
+            type: 'commonJS',
+            rootDir: opts.rootDir
+            //themeFileExtension: '.cssvars.js',
+          },
+          useRemForFontSize: false
```

Comment:
In RSD, we do not set `allowFontScaling: false` on `<Text>`. This means that all text grows and shrinks based on the system font-scaling setting. This is exactly the same behaviour as using `rem` units on the web.

So,
- either we should set this to `true`
- or, we should set `allowFontScaling: false` in RSD and rely on the usage of `rem` units for text that *should* respect font-scaling and manage that manually.